lib/libmbedtls/mbedtls/include/mbedtls/cmac.h

```diff
@@ -55,6 +55,19 @@ struct mbedtls_cmac_context_t
     size_t              unprocessed_len;
 };
 
+/**
+ * \brief               Initialises and allocat cmac context memory
```

Comment:
comment: Initialises and allocat**es** **~~cmac~~CMAC** context memory **.**

Reply:
Yes, I will fix it.